### PR TITLE
Split subscriber and lead generation forms

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-1119)
[this task](https://uktrade.atlassian.net/browse/ED-1097)

# subscribe form
image

# feedback form
image

### CODE REVIEW

File: enrolment/forms.py [hunk 1]
```diff
@@ -26,27 +26,39 @@ class InternationalBuyerForm(forms.Form):
     )
     company_name = forms.CharField(label='Company name')
     country = forms.CharField(label='Country')
+    terms = forms.BooleanField(
+        error_messages={'required': TERMS_CONDITIONS_MESSAGE}
+    )
+
+
+class FeedbackForm(forms.Form):
```
Comment (konradko) [inline enrolment/forms.py]: Is this 'feedback' or lead generation form?

Comment (richtier) [inline enrolment/forms.py]: the feedback form can be used for lead generation, but the form is for general feedback e.g., "this sucks", "improve that", or "I need to find a supplier. can you help?"

Comment (richtier) [inline enrolment/forms.py]: for lead gen. I will rename the form.

File: enrolment/templates/feedback-form.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+				<span class="dont-break-out">Get UK companies to fulfil your business needs</span>
+			</h1>
+		</div>
+	</div>
+	<div class="ed-landing-page-form-inner">
+		<div>
+			Tell us what kind of goods or services you need. We'll put you in touch with relevant UK suppliers.
+		</div>
+		<div class="row-fluid">
+			<form action="{% url 'feedback' %}" method="post" style="margin-top: 30px;">
```
Comment (konradko) [inline enrolment/templates/feedback-form.html]: naming should be consistent, looks like the `feedback` is still referenced in template names, url etc.

File: enrolment/tests/test_forms.py [hunk 3]
```diff
@@ -31,14 +30,37 @@ def test_international_form_accepts_valid_data():
     assert form.is_valid()
 
 
-def test_serialize_international_buyer_forms():
+def test_feedback_form_required():
```
Comment (konradko) [inline enrolment/tests/test_forms.py]: `feedback`

File: ui/context_processors.py [hunk 4]
```diff
@@ -16,11 +16,19 @@ def feature_flags(request):
 def subscribe_form(request):
     return {
         'subscribe': {
-            'form': InternationalBuyerForm(),
+            'form': AnonymousSubscribeForm(),
         },
     }
 
 
+def feedback_form(request):
```
Comment (konradko) [inline ui/context_processors.py]: `feedback` again

File: ui/urls.py [hunk 5]
```diff
@@ -101,10 +102,15 @@
     ),
     url(
         r'^subscribe$',
-        BuyerSubscribeFormView.as_view(),
+        AnonymousSubscribeFormView.as_view(),
         name='subscribe'
     ),
     url(
+        r'^feedback$',
```
Comment (konradko) [inline ui/urls.py]: ditto

**Pull request merged**